oaknational/Oak-Web-Application
# Feat/gleap

## Description

- Adds gleap widget (version 6 -- same as on Acorn -- since 7 doesn't appear to have the `setWidgetUrl` function which we use. I've emailed Lucas about this)
  - updates gleap styles to use our fonts
- Updates consent manager api -- main hook being `useHasConsentedTo/getHasConsentedTo`, which names a service name, e.g. `getHasConsentedTo("gleap") => true|false`
- 

## Issue(s)

Fixes #344 

## How to test

1. Go to https://oak-web-application-53enb4hpm-oak-national-academy.vercel.app/
3. Consent to cookies
4. You should see the gleap widget button in the corner and it should work (should send feedback to the homesite widget in the dashboard)

## Screenshots

![image](https://user-images.githubusercontent.com/12934669/181220321-a016e4b3-4c06-4ffb-8f5c-add9bf8a5c95.png)


## Checklist
